src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,46 @@
+@mixin watch-parameters {
+  height: $watch-radius;
+  width: $watch-radius;
+  border-radius: 50%;
+}
+
+@mixin watch-background {
+  background-image: url(./images/clock.jpg);
+  background-repeat: no-repeat;
+  background-size: $watch-radius + 30;
+  background-position: -17px -10px;
+}
+
+@mixin minute {
+  width: 4px;
```

Comment:
if you created a lot of variables would be great to add this width to variables too. You used it in 3 places